packages/velo-external-db/test/drivers/schema_api_rest_test_support.js

```diff
@@ -9,6 +11,14 @@ const givenCollection = async(name, columns, auth) => {
     }
 }
 
+const retrieveSchemaOperations = async() => {
+    const tempCollectionName = 'tempCollection'
+    await axios.post('/schemas/create', { collectionName: tempCollectionName }, authOwner)
+    const  { schemas }   = (await axios.post('/schemas/find', { schemaIds: [ tempCollectionName ] }, authOwner)).data
```

Comment:
Maybe instead these lines because there are already functions that does that : 

```
await givenCollection('tempCollection',authOwner)
const {schemas} = await retrieveSchemaFor('tempCollection', authOwner).data
...

```